src/Tools/FirstRunCertGenerator/src/Microsoft.AspNetCore.DeveloperCertificates.XPlat.csproj

```diff
@@ -7,6 +7,10 @@
     <PackageTags>aspnet;cli</PackageTags>
 
     <!-- This package contains API for the .NET CLI to generate the aspnet HTTPs dev cert during CLI first-run initialization. -->
+    
+    <!-- Our publishing logic depends on this particular package being NonShipping - if this ever becomes a Shipping project, we will incorrectly calculate the repo's version string,
+         which will cause us to publish installers & checksums to the wrong place. If this ever becomes a Shipping project, we also need to update the following line to reference a different NonShipping project:
+         https://github.com/dotnet/aspnetcore/blob/99c9b8ace1d1d916109882d3f983a26ea4e9b66a/eng/Publishing.props#L38 -->
```

Comment:
Suspect this link will go to a commit that won't be checked in if you squash or rebase. Suggest using `.../master/...#37` because there's no good reason to keep the two commits separate. That link would also avoid pointing to the close comment 😈

Reply:
This commit is the current head of master, not the commit from this PR. There's a chance that publishing.props has some additions/subtractions in master so I wanted to make sure that the link always pointed to the line in question

Reply:
That means the link will always take readers to a commit where the comment you're adding to Publishing.props does not exist ☹️

Reply:
Ah, I guess you're right. I could push a follow-up PR with the correct permalink?

Reply:
Yeah, the more I think about it, the more a "phase 2" makes sense. Pity we can't predict the commit SHA that'll actually go into 'master' without otherwise-unnecessary merge commits or similar.